internal/pkg/template/workload.go

```diff
@@ -998,7 +1005,7 @@ func randomUUIDFunc() (string, error) {
 // envControllerParameters determines which parameters to include in the EnvController template.
 func envControllerParameters(o WorkloadOpts) []string {
 	parameters := []string{}
-	if o.WorkloadType == "Load Balanced Web Service" {
+	if o.WorkloadType == "Load Balanced Web Service" && o.ImportedALB == nil {
```

Comment:
should this be on the `o.ALBEnabled` line below instead? isn't it possible for someone to set the `alias` field even if they have imported an ALB?